src/main/java/com/amazonaws/services/neptune/cluster/AddCloneTask.java

```diff
@@ -378,7 +382,44 @@ private void createInstance(String name,
             requestBuilder = requestBuilder.engineVersion(engineVersion);
         }
 
-        DBInstance targetDbInstance = neptune.createDBInstance(requestBuilder.build()).dbInstance();
+        // Retry configuration
+        int maxRetries = 3;
+        long initialBackoffMillis = 1000; // 1 second
+        DBInstance targetDbInstance = null;
+        CreateDbInstanceRequest request = requestBuilder.build();
+        
+        // Retry loop with exponential backoff
+        for (int attempt = 0; attempt <= maxRetries; attempt++) {
+            try {
+                targetDbInstance = neptune.createDBInstance(request).dbInstance();
+                // If we get here, the request was successful
+                break;
+            } catch (NeptuneException e) {
+                // Check if we've exhausted our retries
+                if (attempt == maxRetries) {
+                    logger.error("Failed to create {} instance after {} attempts", name, maxRetries);
+                    throw e;
+                }
+                
+                // Calculate backoff time with exponential increase and some jitter
+                long backoffMillis = initialBackoffMillis * (long) Math.pow(2, attempt);
+                long jitterMillis = (long) (backoffMillis * 0.2 * Math.random()); // 20% jitter
+                long totalBackoffMillis = backoffMillis + jitterMillis;
+
+                logger.debug("Failed to create {} instance (attempt {} of {}): {}. Retrying...", name, attempt + 1, maxRetries, e.getMessage());
+                
+                try {
+                    Thread.sleep(totalBackoffMillis);
+                } catch (InterruptedException ie) {
+                    Thread.currentThread().interrupt();
+                    throw new RuntimeException("Instance creation interrupted", ie);
+                }
+            }
+        }
+        
+        if (targetDbInstance == null) {
+            throw new RuntimeException("Failed to create DB instance after exhausting all retries");
```

Comment:
Change to logger.warn instead of exception

Reply:
You'll also need to exit this function if you remove the throw. There is logic down below that expects the `targetDbInstance` to exist.